lib/libutee/assert.c

```diff
@@ -41,7 +41,7 @@ void _assert_log(const char *expr __maybe_unused,
 				expr, file, line, func);
 }
 
-void __noreturn _assert_break(void)
+void _assert_break(void)
```

Comment:
> What's the difference between having the attribute at the front compared to at the end?

Well, it looks like sparse expects to find the attribute *before* the function name. Put it after in the .c and you get an error ("error: attributes should be specified before the declarator in a function definition"). And then you have to place it also before in the .h otherwise you get "error: symbol 'pm_panic' redeclared with different type".

I agree it doesn't matter much whether or not we have the attribute on this function, but having it is simpler (no need for `-Wno-suggest-attribute=noreturn`) and more correct anyway.